sros2/sros2/verb/create_key.py

```diff
@@ -26,10 +26,15 @@ class CreateKeyVerb(VerbExtension):
     """Create key."""
 
     def add_arguments(self, parser, cli_name):
-        arg = parser.add_argument('ROOT', help='root path of keystore')
+        arg = parser.add_argument(
+            '-k', '--keystore-root-path',
```

Comment:
To avoid a hard API break, 'ROOT' can be maintained too.
It could be also detected if it was used and print a deprecation warning if later is going to be removed.

P.S.: People will have to read about changes in `sros2` for Foxy, so maybe it's not too bad to break this too.

Reply:
> People will have to read about changes in sros2 for Foxy, so maybe it's not too bad to break this too.

Yeah, I wanted to use this as an opportunity to make the CLI more consistent. The mix use of positional and keyed command arguments was really annoying in terms if UI.

Reply:
Yeah we're talking about cleaning up the non-CLI API for Foxy anyway.

Reply:
:+1: I was considering doing this before Foxy as well, currently only newly introduced verbs like `generate_artifacts` have keyed arguments. For clarity it would be better to have them in a separate PR. If we keep it in this one, can we make this in an independent commit (similar to file renaming commits) ?

Reply:
I will consolidate things in different commits before merging.